kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/BatchesResource.scala

```diff
@@ -82,6 +82,26 @@ private[v1] class BatchesResource extends ApiRequestContext with Logging {
     }
   }
 
+  @ApiResponse(
+    responseCode = "200",
+    content = Array(new Content(
+      mediaType = MediaType.APPLICATION_JSON,
+      schema = new Schema(implementation = classOf[GetBatchesResponse]))),
+    description = "returns the active batch sessions")
+  @GET
+  @Consumes(Array(MediaType.APPLICATION_JSON))
+  def getBatchInfoList(
+      @QueryParam("batchType") batchType: String,
```

Comment:
Shall we return all batch type if user does not specify the batchType ? Otherwise user can not get the toatal batch info list.

Reply:
looks reasonable to me, cc @turboFei

Reply:
sorry, I missed it.

would you like raise a pr to follow up it? @zhaomin1423

Reply:
sure, I do it later.

Reply:
the doc also should be updated.

Reply:
updated